util/fibers/listener_interface.cc

```diff
@@ -34,6 +34,27 @@ auto native_handle(const Connection& conn) {
 using ListType =
     intrusive::list<Connection, Connection::member_hook_t, intrusive::constant_time_size<true>>;
 
+class StdMallocResource : public PMR_NS::memory_resource {
```

Comment:
How is it different from get_default_resource

Reply:
get_default_resource uses new/delete